bigquery/google/cloud/bigquery/table.py

```diff
@@ -1081,6 +1072,46 @@ def _build_schema_resource(fields):
     return infos
 
 
+def _item_to_row(iterator, resource):
+    """Convert a JSON row to the native object.
+
+    .. note::
+
+        This assumes that the ``schema`` attribute has been
+        added to the iterator after being created, which
+        should be done by the caller.
+
+    :type iterator: :class:`~google.cloud.iterator.Iterator`
+    :param iterator: The iterator that is currently in use.
+
+    :type resource: dict
+    :param resource: An item to be converted to a row.
+
+    :rtype: tuple
+    :returns: The next row in the page.
+    """
+    return _row_from_json(resource, iterator.schema)
+
+
+# pylint: disable=unused-argument
```

Comment:
This is for `_rows_page_start`. I commented on the pylint disable instead of the method sig.
The goal would be to make the pylint disable unnecessary.

I'm saying that if it was optional then we won't keep having to disable pylint for it. But probably something better to look at after these are all done.